app/assets/stylesheets/action_page.css.scss

```diff
@@ -266,7 +263,7 @@
   text-align: center;
 
   .privacy-notice {
-    margin-bottom: 0;
+    margin: 10px -10px 0 -10px;
```

Comment:
I couldn't find where this is being rendered. If it's needed and it's possible to do it without negative margins, that would be ideal. I've found that they sometimes cause confusion down the line. Otherwise, not the end of the world.